src/vcbuild/ldfpu.asm

```diff
@@ -182,7 +182,7 @@ FM1:    ; We don't use fprem1 because for some inexplicable
         sahf                            ; transfer to flags
         jp      FM1                     ; continue till ST < ST1
         fstp    ST(1)                   ; leave remainder on stack
-        fstp    tbyte ptr [ecx]
+        fstp    tbyte ptr [rcx]
```

Comment:
Yah, MSVC doesn't allow inline 64-bit assembly. The 32-bit version is inlined.

https://github.com/dlang/dmd/blob/v2.073.0/src/root/longdouble.c#L478